pkg/api/server/cel2sql/select.go

```diff
@@ -35,13 +37,32 @@ func (i *interpreter) translateToJSONAccessors(fieldPath []fmt.Stringer) {
 	fmt.Fprintf(&i.query, ")")
 }
 
-// translateToRecordSummaryColumn
-func (i *interpreter) translateToRecordSummaryColumn(fieldPath []fmt.Stringer) {
-	namer := &schema.NamingStrategy{}
-	switch f := fieldPath[1].(type) {
+func getRawString(s fmt.Stringer) string {
+	switch f := s.(type) {
 	case *Unquoted:
-		fmt.Fprintf(&i.query, "recordsummary_%s", namer.ColumnName("", f.s))
+		return f.s
 	case *SingleQuoted:
-		fmt.Fprintf(&i.query, "recordsummary_%s", namer.ColumnName("", f.s))
+		return f.s
+	}
+	return s.String()
+}
+
+// translateIntoStruct
+func (i *interpreter) translateIntoStruct(fieldPath []fmt.Stringer) error {
+	namer := &schema.NamingStrategy{}
+	rawSql := getRawString(fieldPath[0])
+	rawField := getRawString(fieldPath[1])
+	sqlTemplate, err := template.New("").Parse(rawSql)
```

Comment:
Are there template placeholders being interpolated somewhere?

Reply:
Yes, the summary is configured like this:

```
"summary": {
	...
	SQL:        `recordsummary_{{.Field}}`,
},
```